content/_data/upgrades/2-204-5.adoc

```diff
@@ -1,4 +1,7 @@
 ==== Custom System Logging Formats Ignored
 
 Custom system logging formats are ignored in Jenkins 2.204.5 as a result of link:https://issues.jenkins-ci.org/browse/JENKINS-57888[JENKINS-57888].
-Fixes for the high default maximum form size limit and reverse proxy redirection are included in Jenkins 2.204.5.
+Fixes for the high default maximum form size limit and reverse proxy redirection are included in Jenkins 2.204.5, and they required downgrade of the 
```

Comment:
Needs a completion of the sentence fragment.  Not sure what was intended, made my guess.
```suggestion
Fixes for the high default maximum form size limit and reverse proxy redirection are included in Jenkins 2.204.5, and they required downgrade of the Jetty dependency.
```